source/anti-patterns/language.md

```diff
@@ -100,7 +90,7 @@ In most cases, `$NIX_PATH` is set to the latest channel when Nix is installed, a
 
 :::{note}
 [Channels](https://nixos.wiki/wiki/Nix_channels) are a way of distributing Nix software.
-They are being phased out, but still used by default.
+They are being phased out by flakes, but still used by default.
```

Comment:
We should first explain visibly what flakes are and how to use them before telling people "channels are all over the place, but forget that. use flakes, but you'll have to figure out what they even do on your own."

So we would only refer to flakes once we have an authoritative (and hopefully satisfactory) definition. There are  RFC49 (closed!) and Eelco's blog posts, and I'm fine with linking those, but something with Nix™ on them would be better.